t/categories/99-problems.t

```diff
@@ -481,7 +482,6 @@ sub expected-output {
     %expected-output{"P39-rhebus.pl"} = q:to/END/;
     11 13 17 19
     3 5 17 257 65537
-    1 2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71 73 79 83 89 97
```

Comment:
Why does this output no longer appear when running P39-rhebus.pl?  It used to appear in earlier Rakudo versions, hence this looks like a bug in the example.